torch/fx/symbolic_trace.py

```diff
@@ -717,6 +719,10 @@ def my_custom_function(x, y):
     _wrapped_fns_to_patch.append((f.f_globals, fn_name))
     return fn_or_name
 
+# HACK: `torch.fx.symbolic_trace` refers to the function, not the module
+# wrap's default `__module__` is `torch.fx.symbolic_trace`. This leads to fuckery
+wrap.__module__ = 'torch.fx'
```

Comment:
I think the real problem here is that this file (`symbolic_trace.py`) shadows the symbol that `torch/__init__.py` exports from this file (`symbolic_trace`). This is probably bad news bears in general (e.g. `import torch.fx.symbolic_trace` and `from torch.fx import symbolic_trace` will do different things!), I would suggest making this file `_symbolic_trace.py` instead. Or if you don't want it to be hidden, `symbolic_tracer.py` maybe?